identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -196,6 +196,7 @@ invalid.callback.url.found.in.the.request=Invalid callback URL found in the requ
 select.language=Select Language
 username.length.should.be=The username length should be between
 username.with.symbols=The username should consist of alphanumeric characters (a-z, A-Z, 0-9) and must include at least one letter.
+username.with.special.character.symbols=The username must include at least one letter and should not consist of invalid special characters.
```

Comment:
Would users be informed what are the invalid special characters?

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/4790/commits/cd880a4bc3295c222c65b70c471cba50594c172f